demisto_sdk/commands/generate_outputs/README.md

```diff
@@ -0,0 +1,86 @@
+## generate-outputs
+Generate outputs for an integration.
+Previously `json-to-outputs` and `generate-context` combined.
+
+**Use-Cases**
+This command is used to generate context paths automatically from an example file directly into an integration yml file.
+Also supports converting JSON format to demisto entry context yaml format.
+
+**Arguments**:
+* *-c, --command*
+  Command name (e.g. xdr-get-incidents)
+* *-i, --input*
```

Comment:
think you meant the -j --json here (if we decide to keep this argument and not merge to -i as questioned)